bedrock/firefox/templates/firefox/features/pdf-complete-fr.html

```diff
@@ -0,0 +1,94 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "firefox/features/base-article.html" %}
+
+
+{% block article_title_short %}Remplir un PDF en ligne: fiche d’impôts{% endblock %}
+{% block article_title %}Remplir un PDF en ligne : fiche d’impôts{% endblock %}
+{% block page_desc %}Retrouvez un guide pas-à-pas pour remplir et même corriger sa déclaration d’impôts, en utilisant les options intégrées de Firefox.{% endblock %}
+
+{% block article_content %}
+
+<p>Fini les soucis liés à la paperasse, place à la simplicité administrative !  Que vous ayez besoin de remplir une première déclaration, de corriger une erreur ou de gérer vos justificatifs, Firefox vous offre une solution rapide et sécurisée pour vos données. </p>
+
+<p>Remplir des PDF n'a jamais été aussi simple et fiable, même quand on parle de fiches d’impôts.</p>
+
+<h2>Remplir sa PDF</h2>
+
+<p>Renseigner les informations de manière électronique plutôt que de le faire à la main pour éviter les erreurs, les reprises et un procédé administratif qui traine. C’est d’autant plus à portée de main si c’est votre première déclaration fiscale&nbsp;! </p>
+
+<ol class="mzp-u-list-styled">
+  <li>
+    <p><strong>Ouvrir le document :</strong> une fois le formulaire vierge téléchargé, faites un clic droit dessus et sélectionnez "Ouvrir avec Firefox". Le document apparaît dans un onglet de votre navigateur.</p>
+    <p>
+      {{ resp_img(
+        url="img/firefox/features/edit-pdf/fr-edit-pdf-ouvrir.png",
+        srcset={
+            "img/firefox/features/edit-pdf/fr-edit-pdf-ouvrir-high-res.png": "1.5x"
+        },
+        optional_attributes={
+          "alt": "",
+          "width": "600"
+        }
+      ) }}
+    </p>
+  </li>
+  <li>
+    <p><strong>Renseigner vos informations :</strong> Cochez  les cases appropriées à l’aide de l’outil Dessin symbolisé par un crayon.</p>
+  </li>
+  <li>
+    <p><strong>Pour ajouter du texte :</strong> cliquez sur l'icône Texte pour ajouter du texte tels que votre numéro fiscal, vos noms, adresse et autre renseignements fiscaux. </p>
+    <p>
+      {{ resp_img(
+        url="img/firefox/features/edit-pdf/fr-edit-pdf-texte.png",
+        srcset={
+            "img/firefox/features/edit-pdf/fr-edit-pdf-texte-high-res.png": "1.5x"
+        },
+        optional_attributes={
+          "alt": "",
+          "width": "600"
+        }
+      ) }}
+    </p>
+  </li>
+  <li>
+    <p>Vous pouvez déplacer n’importe quel de vos blocs de texte et ajuster la taille, comme bon vous semble. Vous pouvez facilement passer d’une page à l’autre de votre déclaration ou avoir une vue d’ensemble en sélectionnant dans les Outils. </p>
```

Comment:
**Nit:** (again based on copy lol) it doesn't seem this should be a list item?

Reply:
✅